pkg/minikube/constants/constants.go

```diff
@@ -38,6 +38,8 @@ const MiniShiftEnvPrefix = "MINISHIFT"
 // MiniShiftHomeEnv is the environment variable used to change the Minishift home directory
 const MiniShiftHomeEnv = "MINISHIFT_HOME"
 
+const SupportedOpenshiftTag = "1.3.3"
```

Comment:
MinSupportedOpenshiftTag, maybe? We support anything which comes above, right? Or you could add '>' here as well, then you would not need the Sprintf